pkg/utils/utils.go

```diff
@@ -265,3 +270,69 @@ func GetArgs(command string, frameWork string, frameworkVersion int,
 
 	return args
 }
+
+// Read locators from the file and convert it into array of locator config
+func ExtractLocators(locatorFilePath, flakyTestAlgo string, logger lumber.Logger) ([]core.LocatorConfig, error) {
+	locatorArrTemp := []core.LocatorConfig{}
+	inputLocatorConfigTemp := &core.InputLocatorConfig{}
+
+	if flakyTestAlgo == core.RunningXTimesShuffle {
+		content, err := os.ReadFile(locatorFilePath)
+		if err != nil {
+			logger.Errorf("Error when opening file: ", err)
```

Comment:
Missing `%v` in log statement for err (does it work in the way it is written with Errorf ?). Also we should add more parameters in log message. And in standard practice log messages start from small letter.